src/AccessibilityInsights.SharedUx/Controls/CustomControls/CustomExpander.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System.Windows.Automation.Peers;
+using System.Windows.Controls;
+
+namespace AccessibilityInsights.SharedUx.Controls.CustomControls
+{
+    public class CustomExpander : Expander
+    {
+        protected override AutomationPeer OnCreateAutomationPeer()
+        {
+            return new CustomExpanderAutomationPeer(this);
+        }
+    }
+
+    public class CustomExpanderAutomationPeer : ExpanderAutomationPeer
+    {
+        GroupItem parentGroupItem;
+
+        public CustomExpanderAutomationPeer(Expander owner) : base(owner) => parentGroupItem = owner?.TemplatedParent as GroupItem;
+
+        /// <summary>
+        /// In .NET 4.8 grouped ListViews, the GroupItem's HasKeyboardFocus value is set based on its child expander's
```

Comment:
Thanks for the explanation. What's our risk if the .NET team fix their behavior in the future?

Reply:
I see a low risk that this change negatively impacts future behavior. There is more of a risk that something is changed which renders this obsolete/ineffectual--I have no way of judging the likelihood of that scenario. We'd just want to pay special attention to this ListView in future .NET updates, which is already the case.